include/ghex/unstructured/communication_object_ipr.hpp

```diff
@@ -376,7 +386,7 @@ namespace gridtools {
                              typename Memory,
                              typename Field,
                              typename O>
-                    void allocate(Memory& mem,
+                    void allocate(Memory& mem, // reference not strictly needed here
```

Comment:
why is this comment useful? could you expand it?

Reply:
It's just a reminder, memory is already a pointer, and it is used as such inside, so a copy could also be ok. Same happens in comm2. I just did not want to change much the interface with respect of the comm2 one, since that should be harmless.

Reply:
I pushed some documentation and removed the pass by reference from this function (allocate), in which it is actually not needed.

Reply:
From my side it should be ok now